third_party/nvfuser/csrc/type.h

```diff
@@ -50,7 +50,8 @@ enum class PredicateType {
   Misaligned,
   Shift,
   Padding,
-  ReductionWrite
+  ReductionWrite,
+  LoopRotation
```

Comment:
Please add a comment on `LoopRotation`. See above for the comments on the existing types.

Reply:
Added